### PR TITLE
Update index.html

### LINKED ISSUE #552: Replace Google Analytics Tracking with Matomo
## Is your feature request related to a problem? Please describe.
We are switching our web analytics to use Matomo Analytics, so we need to replace the Google tracking code with the Matomo Tag Manager ID and code.

## Describe the solution you'd like
Replace the Google Analytics tracking code with Matomo Tag Manager tracking code in `public/index.html` (see PR #287 for previous Google Tag Manager implementation). 

## Additional context
Note that there was some troubleshooting in dimes PR #349  to make Google Tag Manager page titles work correctly, so review and test this implementation with Matomo.

### PR DESCRIPTION
**Pull request by EcZX**
Replaced Google Analytics Tracking Code with Matomo Tracking Code

### CODE REVIEW

**Comment by HaSistrunk** [comment]
Hey @EcZX , it looks like you actually made changes in the development branch and are trying to merge them into the issue-552 branch instead of the other way around.

Let me know if you need help reverting that and making the changes in the issue branch.

File: public/index.html [hunk 1]
```diff
@@ -29,13 +29,14 @@
...
+<script>
+var _mtm = window._mtm = window._mtm || [];
+_mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+g.async=true; g.src='https://cdn.matomo.cloud/rockarch.matomo.cloud/container_wRUBz58c.js'; s.parentNode.insertBefore(g,s);
+</script>
+<!-- End Matomo Tag Manager -->
   </head>
   <body>
     <!-- Google Tag Manager (noscript) -->
```
Comment (HaSistrunk) [inline public/index.html:42]: You also need to remove this Google Tag Manager (noscript) in the `<body>`

**Comment by EcZX** [comment]
Thanks @HaSistrunk, should i close this pull request and create a new one?

**Comment by HaSistrunk** [comment]
> Thanks @HaSistrunk, should i close this pull request and create a new one?

Yeah, I reverted that last commit so you can:
1. Edit the comment in this pull request (PR) to remove the text "fix # 552", so that when you close the PR it doesn't close the issue.
2. Close this PR
3. Make the necessary changes to the file in the Issue-552 branch
4. Reopen a PR with a request to merge `Issue-552` into `development`

**Pull request closed without merging**